go/libkb/sig_chain.go

```diff
@@ -343,6 +348,12 @@ func (sc *SigChain) GetCurrentSubchain(eldest keybase1.KID) (links []*ChainLink,
 	l := len(sc.chainLinks)
 	lastGood := l
 	for i := l - 1; i >= 0; i-- {
+
+		if sc.chainLinks[i].IsStubbed() {
+			lastGood = i
+			continue
+		}
```

Comment:
ok, just break out if the stubbed link is an eldest link?  otherwise, these links shouldn't be doing implicit eldest changes, since they're modern